.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -313,7 +321,10 @@ jobs:
     # Build Quantum and Gradient Dialects
     - name: Build MLIR Dialects
       run: |
-        export PATH=$GITHUB_WORKSPACE/llvm-build/bin:$PATH
+        python3.9 -m pip install ninja
+        rm -f /usr/bin/ninja
+        ln -s /opt/python/cp39-cp39/bin/ninja /usr/bin/ninja
+        export PATH=/opt/rh/gcc-toolset-13/root/usr/bin:$GITHUB_WORKSPACE/llvm-build/bin:$PATH
```

Comment:
Maybe the same here for the LD_LIBRARY_PATH, which on my container looks like:
```
LD_LIBRARY_PATH=/opt/rh/gcc-toolset-13/root/usr/lib64:/opt/rh/gcc-toolset-13/root/usr/lib:/opt/rh/gcc-toolset-12/root/usr/lib64:/opt/rh/gcc-toolset-12/root/usr/lib:/opt/rh/gcc-toolset-12/root/usr/lib64/dyninst:/opt/rh/gcc-toolset-12/root/usr/lib/dyninst
```

Reply:
Done